pyds8k/base.py

```diff
@@ -128,7 +128,8 @@ def one(self, route, resource_id, rebuild_url=False):
             get_resource_by_route = \
                 import_module('{}.resources.utils'.format(__package__)
                               ).get_resource_by_route
-        url = self._set_url(route, resource_id, rebuild_url=rebuild_url)
+        url = self._set_url(route.split('.')[-1],
```

Comment:
what is the reason to change it? is it related to the new api endpoint?

Reply:
This “route” will be used for url split and getting CSPPRC class object. Because previous api is like /v1/XXXX instead of /v1/cs/XXXX, route will be used differently in url split and getting CSPPRC class object. Url split needs pprcs only and getting CSPPRC class object needs cs.pprcs.